Lesson_2_JSON_YAML_XML/house.json

```diff
@@ -0,0 +1,48 @@
+{
+    "house": {
+        "rooms": [
+            {
+                "kitchen": {
+                    "table": 1,
+                    "chairs": 5,
+                    "oven": 1,
+                    "cleaned": true,
+                    "state": "redecorated"
+                }
+            },
+            {
+                "livingroom": {
+                    "table": 0,
+                    "chairs": 2,
+                    "TV set": 1,
+                    "cleaned": false,
+                    "state": "shabby"
+                }
+            },
+            {
+                "bedroom": {
+                    "table": 0,
+                    "chairs": 0,
+                    "bed": 1,
+                    "TV set": 1,
+                    "cleaned": true,
+                    "state": "shabby"
+                }
+            }
+        ],
+        "owner": [
```

Comment:
looks good, but `owners` might fit better as it will implicitly say it's an array of items

Reply:
changed, thanks